### PR TITLE
fix(gemini-ics): Increase loader and db instance types

### PR DESCRIPTION
**Pull request by aleksbykov**
Gemini could generate very complex schemas and this could lead that OOM on loader and not enough resouces on db nodes.

Increase instance types to avoid unpredictable failures

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by fgelcer — APPROVED** [review]
a bit more context to whoever will review this PR would be helpful... LGTM

**Comment by fgelcer** [comment]
@aleksbykov , can you link here the staging job you are running?

**Comment by aleksbykov** [comment]
https://jenkins.scylladb.com/view/enterprise-2023.1/job/enterprise-2023.1/job/Reproducers/job/gemini-3h-ics-cdc-with-nemesis-test/
https://jenkins.scylladb.com/view/enterprise-2023.1/job/enterprise-2023.1/job/Reproducers/job/gemini-3h-ics-test/

**Comment by aleksbykov** [comment]
1 reproducer is passed.
2nd failed with gemini error. We can merge the request

File: test-cases/gemini/gemini-basic-3h-ics.yaml [hunk 1]
```diff
@@ -3,22 +3,26 @@ n_db_nodes: 3
...
 
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d --duration 10800s --warmup 1800s -c 50 -m mixed -f --non-interactive \
+gemini_cmd: "gemini -d --duration 10800s --warmup 1800s -c 25 -m mixed -f --non-interactive \
 --cql-features normal --table-options \"compaction={'class': 'IncrementalCompactionStrategy'}\" \
---max-mutation-retries 5 --max-mutation-retries-backoff 500ms \
---async-objects-stabilization-attempts 5 --async-objects-stabilization-backoff 500ms "
+--max-mutation-retries 25 --max-mutation-retries-backoff 10s \
```
Comment (soyacz) [inline test-cases/gemini/gemini-basic-3h-ics.yaml:17]: these number are pretty big. I propose setting it to 10 and 2s (unless you tested and proven current ones are right)

Comment (aleksbykov) [reply test-cases/gemini/gemini-basic-3h-ics.yaml:17]: For that moment this numbers give most stable results, especially for MV and SI operations

**Review by soyacz — APPROVED** [review]
LGTM

**Review by vponomaryov — APPROVED** [review]
LGTM

@roydahan this one is waiting for you.

**Review by roydahan — APPROVED** [review]
I don't like it very much, but ok for now, till we decide how to refactor our Gemini tests.

**Comment by roydahan** [comment]
Just noticed this PR is directly to branch-2023.1?
Why is it?
it should go through master

**Comment by aleksbykov** [comment]
@roydahan , the problem that enterprise 2021.1 doesn't support i4i. and also master already switched to i4i, while enterprise is still use i3. So i created directly to 2023.1

**Review by temichus — APPROVED** [review]

**Pull request merged**